packages/server/src/asyncQuestion/asyncQuestion.service.ts

```diff
@@ -115,6 +138,31 @@ export class AsyncQuestionService {
         }
       });
     });
+    // FEED alerts to all participants who commented (excluding current commenter and creator), regardless of email subscriptions
+    const participantIds = Array.from(
+      new Set(
+        updatedQuestion.comments
+          .map((c) => c.creator.id)
+          .filter((id) => id !== commenter.id && id !== questionCreatorId),
+      ),
+    );
+    await Promise.all(
+      participantIds.map((uid) =>
+        AlertModel.create({
+          alertType: AlertType.ASYNC_QUESTION_UPDATE,
+          deliveryMode: AlertDeliveryMode.FEED,
+          sent: new Date(),
+          userId: uid,
+          courseId: updatedQuestion.courseId,
+          payload: {
+            courseId: updatedQuestion.courseId,
+            questionId: updatedQuestion.id,
+            subtype: 'commentOnOthersPost',
+            summary: `${commenterIsStaff ? commenter.name : 'Someone'} commented on an Anytime Question you follow`,
```

Comment:
```suggestion
            summary: `${commenterIsStaff ? commenter.name : 'Someone'} commented on an Anytime Question you commented on`,
```

Don't like using the term "follow" here since people might think they can "unfollow" it